cassdegrees/templates/managecourses.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+
+{% block page-title %}Manage Courses{% endblock %}
+
+{% block subtitle %}Add, Modify or Delete Courses{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="/manage_courses/" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Add Course</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="id">Course Name:</label>
+                <input class="text" id="text" type="text" name="name" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="id">Course Code:</label>
+                <input class="text" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Year:</label>
+                <!-- static min max values are only temporary -->
+                <input class="text" id="number" type="number" name="year" min="1990" max="2100" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Semester(s):</label>
+                <select name="semester[]" size="2" multiple>
+                    <option value="semester1">Semester 1</option>
+                    <option value="semester2">Semester 2</option>
+                </select>
+            </p>
+
+             <p>
+                <label for="text">Units:</label>
+                 <!-- max value is chosen arbitrarily -->
+                <input class="text" id="number" type="number" name="units" min="6" max="48" step="6" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Requisite Course(s):</label>
+                <select name="semester[]" size="5" multiple>
+                    {% for course in courses %}
+                        <option value="{{ course.code }}_{{ course.year }}">{{ course.code }}_{{ course.year }}</option>
```

Comment:
```suggestion
                        <option value="{{ course.code }}">{{ course.code }}</option>
```

Course year is not relevant to requisites, so I have removed them (Need to ensure this is also done when passing the API to prevent repeat values). This also raises another important issue, which is that the requisites is not currently represented in the database. This may be a problem for @aidengeorge97 to solve